### PR TITLE
add nuxt-google-sheets-parser

### PR DESCRIPTION
**Pull request by fureweb-com**
@Geminii recommended me to add this module here, so I make pull request here.

### CODE REVIEW

File: modules/nuxt-google-sheets-parser.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: nuxt-google-sheets-parser
+description: Public google sheets parser for nuxt.js
+repo: fureweb-com/nuxt-google-sheets-parser
+npm: nuxt-google-sheets-parser
+icon: nuxt-google-sheets-parser.svg
+github: 'https://github.com/fureweb-com/nuxt-google-sheets-parser'
+website: 'https://github.com/fureweb-com/nuxt-google-sheets-parser'
+learn_more: 'https://github.com/fureweb-com/nuxt-google-sheets-parser'
+category: 'Libraries'
```
Comment (Geminii) [inline modules/nuxt-google-sheets-parser.yml:9]: Single Quote not needed for category, just : **Libraries**

File: static/icons/nuxt-google-sheets-parser.svg [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Geminii) [inline static/icons/nuxt-google-sheets-parser.svg:1]: xml definition not necessary here with Adobe illustrator comment too ^^

File: static/icons/nuxt-google-sheets-parser.svg [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+<!-- Generator: Adobe Illustrator 25.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 64 88" style="enable-background:new 0 0 64 88;" xml:space="preserve">
```
Comment (Geminii) [inline static/icons/nuxt-google-sheets-parser.svg:3]: I recommend you to have a svg directly in 40*40px because it will be reduce at this size otherwise

**Review by Geminii — COMMENTED** [review]
Great module which will serve for the community 🔥

File: static/icons/nuxt-google-sheets-parser.svg (hunk 3 above)
Comment (fureweb-com) [reply static/icons/nuxt-google-sheets-parser.svg]: I really thanks for your help here, too!
I also fix this line but honestly I don't know about svg attributes properly.
But I could see this logo's container sizes 40x40 on browser.
If I was wrong, I'll fix it.
Thanks Jimmy!

**Comment by pi0** [comment]
Hi @fureweb-com Thanks so much for creating PR and [public-google-sheets-parser](https://github.com/fureweb-com/public-google-sheets-parser) seems an amazing package

 But checking [module code](https://github.com/fureweb-com/nuxt-google-sheets-parser/blob/main/lib/plugin.js) it doesn't seem to be eligible be a nuxt module. Since utilities like google-sheets-parser might be imported by used in pages that use them. (otherwise we add extra size to main chunk)
 
 It would be nice explaining how to use with nuxt in main package like this:

```js
export default {
  created() {
    this.gsparser = new PublicGoogleSheetsParser() 
  }
  methods: {
    parseSheet (sheetId, sheetName = null) {
      return this.gsparser.parse(sheetId, sheetName)
    }
  },
  async mounted () {
    // see: https://docs.google.com/spreadsheets/d/10WDbAPAY7Xl5DT36VuMheTPTTpqx9x0C5sDCnh4BGps
    const sheetId = '10WDbAPAY7Xl5DT36VuMheTPTTpqx9x0C5sDCnh4BGps'

    const data1 = await this.parseSheet(sheetId)
    console.log(data1) // [{"a":1,"b":2,"c":3},{"a":4,"b":5,"c":6},{"a":7,"b":8,"c":9}]

    const data2 = await this.parseSheet(sheetId, 'Sheet2')
    console.log(data2) // [{"a":10,"b":20,"c":30},{"a":40,"b":50,"c":60},{"a":70,"b":80,"c":90}]
  },
  // ...
}
```

At the moment modules website is indented for modules only.  But we plan to add an integrations section to the website to mention solutions like this.

**Comment by fureweb-com** [comment]
Thank you for the kind explanation!

**Pull request closed without merging**